### PR TITLE
chore: Convert KMManager to Markdown

### PR DESCRIPTION
**Pull request by darcywong00**
Follow-on to #1267 

Prepares `/developer/engine/android/18.0/KMManager` for API updates from keymanapp/keyman#11436

### CODE REVIEW

File: developer/engine/android/18.0/KMManager/index.md [hunk 1]
```diff
@@ -0,0 +1,267 @@
...
+: returns whether the help bubble is enabled
+
+[`isKeyboardLoaded()`](isKeyboardLoaded)
+: returns whether the specified in-app or system keyboard is loaded
+
+[`keyboardExists()`](keyboardExists)
+: returns whether the specified keyboard exists in keyboards list
+
+`lexicalModelExists()`
+: returns whether the specified lexical model exists in lexical models list
```
Comment (mcdurdin) [inline developer/engine/android/18.0/KMManager/index.md:171]: Is this missing a separate page?

**Review by mcdurdin — APPROVED** [review]

File: developer/engine/android/18.0/KMManager/index.md (hunk 1 above)
Comment (darcywong00) [reply developer/engine/android/18.0/KMManager/index.md:171]: Yeah, some of the API's from previous releases never got fully documented..

Comment (mcdurdin) [reply developer/engine/android/18.0/KMManager/index.md:171]: Can we add those now while we are doing documentation?

Comment (darcywong00) [reply developer/engine/android/18.0/KMManager/index.md:171]: I'll add that to #1270

**Pull request merged**